test/services/UIBuilderSpec.scala

```diff
@@ -354,7 +354,87 @@ class UIBuilderSpec extends BaseSpec with ProcessJson {
     val headingText = Text(headingPhrase.langs)
   }
 
+  trait NumberListTest extends Test {
+    val headingPhrase = Phrase(Vector("Heading", "Heading"))
+    val num1Phrase = Phrase(Vector("Line1", "Welsh Line1"))
+    val num2Phrase = Phrase(Vector("Line1", "Welsh Line1"))
+    val num3Phrase = Phrase(Vector("Line1", "Welsh Line1"))
+    val num4Phrase = Phrase(Vector("Line1", "Welsh Line1"))
```

Comment:
Doesn't look like these phrase values get used anywhere - just set on the callouts - all say Line1 - could use a single phrase if no checking done in tests of the phrase order

Reply:
updated